src/table.rs

```diff
@@ -361,7 +431,10 @@ impl Column {
                                     if let DescriptionElement::UInt(value) = v {
                                         (
                                             DescriptionElement::Enum(
-                                                reverse_map.get(value).unwrap().to_string(),
+                                                reverse_map
+                                                    .get(value)
+                                                    .unwrap_or(&"_NO_MAP_".to_string())
```

Comment:
This calls `to_string` twice for `"_NO_MAP_"`: one in `unwrap_or` and another in the next line, Line 437. Please combine Lines 436-437 into `map_or`.

Reply:
Changed.